.infrastructure/03_s3.tf

```
@@ -0,0 +1,26 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+}
+
+resource "aws_s3_bucket" "cms_medialibrary_bucket" {
+  bucket = "media-library-s3-strapi-${local.account_id}"
```

```suggestion
  bucket = "strapi-medialibrary-${local.account_id}"
```
WDYT? I don't think it gives a more info the `s3` within the name.
Moreover, I'd use the `medialibrary` instead of `media-library` just to be consistent with the resource's name.

---

yes